src/dlsnode/neo/request/Put.d

```diff
@@ -139,14 +139,8 @@ private scope class PutImpl_v0: PutProtocol_v0
 
     override protected bool prepareChannel ( cstring channel_name )
     {
```

Comment:
Actually, it's probably better to just remove these requests from the map of request handlers. That way, the neo protocol will automatically send back a "not supported" status code to the client.